Makefile

```diff
@@ -9,10 +9,8 @@ PROJ = google_containers
 
 PKG_SOURCES := $(shell find pkg -name '*.go')
 
-node-problem-detector: ./bin/node-problem-detector
-
-./bin/node-problem-detector: $(PKG_SOURCES) node_problem_detector.go
-	CGO_ENABLED=0 GOOS=linux godep go build -a -installsuffix cgo -ldflags '-w' -o node-problem-detector
+node-problem-detector: $(PKG_SOURCES) node_problem_detector.go
+	GOOS=linux go build -ldflags '-w -extldflags "-static"' -o node-problem-detector
```

Comment:
@Random-Liu I have manually download #49. And compile it in a machine where `libsystemd-dev` has been downloaded. I run it well in a machine where `libsystemd-dev` has not been installed. 

Seems i can not reproduce it.